<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CServer</class>
 <widget class="QMainWindow" name="CServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>970</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>149</y>
      <width>871</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>821</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_5">
       <property name="minimumSize">
        <size>
         <width>1131</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>730</y>
          <width>131</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>一键识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>730</y>
          <width>131</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>详细识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>730</y>
          <width>131</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>全选</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QTableView" name="Pic_View_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>80</y>
          <width>320</width>
          <height>420</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>520</y>
          <width>75</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 101, 255);</string>
        </property>
        <property name="text">
         <string>识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>745</x>
          <y>520</y>
          <width>75</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 101, 255);</string>
        </property>
        <property name="text">
         <string>全部识别</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>375</x>
          <y>520</y>
          <width>75</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 101, 255);</string>
        </property>
        <property name="text">
         <string>下一张</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>440</y>
          <width>140</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="Card_View">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>400</width>
          <height>300</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>440</y>
          <width>215</width>
          <height>70</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>图像预览</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>车牌区域</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>400</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>车牌号：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>文件列表：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="Upload_Button">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>525</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(3, 169, 244);
border-radius:5px;
color:white;</string>
      </property>
      <property name="text">
       <string>上传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete_Button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>525</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(3, 169, 244);
border-radius:5px;
color:white;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Allpick_Button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>525</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(3, 169, 244);
border-radius:5px;
color:white;</string>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
     <widget class="QTableView" name="Succeess_View">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>55</y>
        <width>380</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="Fail_View">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>55</y>
        <width>380</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(122, 122, 122);</string>
      </property>
      <property name="text">
       <string>可识别列表:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(122, 122, 122);</string>
      </property>
      <property name="text">
       <string>未能识别列表:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>155</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>按车牌号查询</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>155</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按身份证查询</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按车主姓名查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Number_Label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>   输入</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2016</year>
        <month>8</month>
        <day>12</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(103, 103, 103);</string>
      </property>
      <property name="text">
       <string>查询结果：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>20</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(128, 128, 128);</string>
      </property>
      <property name="text">
       <string>至</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2016</year>
        <month>8</month>
        <day>12</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="Check_Button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/cha.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="Info_View">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>750</width>
        <height>330</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>330</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>750</width>
        <height>330</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="midLineWidth">
       <number>20</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>154</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 169, 244);
border-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>服务器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>现代化智能交通监控系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/tou.png);
image: url(:/tou.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>欢迎您，XXX</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>154</y>
      <width>140</width>
      <height>596</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0288D1;
border-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recognization_Button">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>170</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>识别(&amp;R)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit_Button">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>225</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>提交(&amp;S)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Query_Button">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>280</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查询(&amp;Q)</string>
    </property>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>graphicsView_4</zorder>
   <zorder>Recognization_Button</zorder>
   <zorder>Submit_Button</zorder>
   <zorder>Query_Button</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFd">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action1_4"/>
    <addaction name="action2_6"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="Usermanage_action"/>
    <addaction name="separator"/>
    <addaction name="action2_4"/>
    <addaction name="Exit_action"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuFd"/>
  </widget>
  <action name="actionT">
   <property name="text">
    <string>添加(A)</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>设置IP(&amp;S)</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action2_4">
   <property name="text">
    <string>重新连接(&amp;R)</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="Exit_action">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>添加用户(&amp;A)</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="Usermanage_action">
   <property name="text">
    <string>用户管理(&amp;M)</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action2_6">
   <property name="text">
    <string>说明文档(&amp;I)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
